### DISCUSSION TITLE
Failed to get binary client for engine errors

### DISCUSSION THREAD

**Discussion by Stefan-Balta** [longhorn/longhorn#6580] [Q&A]
`longhorn-manager` pods emit logs of the form:
`Failed to get engine proxy of pvc-RANDOM_CHARS-e-d80d9fd1 for 
volume pvc-RANDOM_CHARS" collector=volume 
error="failed to get binary client for engine pvc-RANDOM_CHARS-e-d80d9fd1: 
cannot get client for engine pvc-RANDOM_CHARS-e-d80d9fd1: engine is not running" node=SOME_NODE`

Has anyone seen this error before? What could cause it?

**Comment by derekbit** [comment]
What's the engine status of the volume? You can check it by
```
kubectl -n lobnghorn-system get engines
```

**Reply by Stefan-Balta** [reply]
Every engine from error logs that I checked is stopped.

**Reply by derekbit** [reply]
Can you provide a support bundle?

**Reply by Stefan-Balta** [reply]
Unfortunately, due to our company's policy, we cannot share a durmp from production clusters (that also weighs over 1 GB). We have noticed that these logs mention only detached volumes.

**Reply by derekbit** [reply]
OK. You can check the longhorn-manager and instance-manager logs and see why the engines are stopped.

**Reply by DraganDjuricOB** [reply]
These logs are present in the v1.6.1 as well.
I have checked the state of the volumes and found that all of them are detached @derekbit 

Should this be logged as an error? Is this the expected behavior? 
And if so, can we just ignore these logs?
```
$ kubectl -n longhorn-system get engines | grep stopped | wc -l
100
```
![image](https://github.com/longhorn/longhorn/assets/106532613/49b1f21f-a9bb-484c-8654-f9ccca998bfa)
